binrz/man/rz-gg.1

```diff
@@ -16,87 +17,88 @@
 .Op Fl B Ar hexpairs
 .Op Fl c Ar k=v
 .Op Fl C Ar file
-.Op Fl n Ar num32
-.Op Fl N Ar num64
+.Op Fl n Ar dword
+.Op Fl N Ar dword
```

Comment:
revert these.

Reply:
please submit a patch for the help command of this tool and change dword to `num64` and `num32`